### PR TITLE
DOC-7652: Document how to manage channels via the System Console

### LINKED ISSUE #7652: Document how to manage channels via the System Console
## Context

Details on how to manage Mattermost channels as a system admin using the System Console are missing from the Mattermost Product Documentation.

## Docs Request

Develop admin documentation for the following topics:

- Find channels
- Filter channel searches
- Review channel configuration
- Advanced access controls
  - Create messages
  - Message reactions
  - Manage members
  - Channel mentions
  - Manage bookmarks
- Channel management
  - Sync group members
  - Public channel or private channel
- Groups (Skipped)
- Members
  - Add members
  - Remove members

**Note**: The ability to add groups to a channel also needs to be documented, but requires additional internal investigation/testing first.

## How you can help

- Create a single docs PR that contains all of these changes across as many commits as necessary
- Use the latest Mattermost mobile app and the standard Mattermost Denim theme to capture all workflows.
- Use existing admin docs content as a starting point for voice and tone guidance. The Docs team will review and provide support for publication.

### PR DESCRIPTION
**Pull request by Rutam21**
## Summary

Detailed steps for the channel administration workflows were required but absent. So, this PR aims to add the steps for all those workflows for channel management on the [User management configuration settings](https://docs.mattermost.com/configure/user-management-configuration-settings.html#channels) page. 

The following workflows have been added.

- Find Channels
- Filter channel searches
- Review channel configuration
- Advanced Access Controls
     - Create Posts
     - Post Reactions
     - Manage Members
     - Channel Mentions
     - Manage Bookmarks
- Channel Management
     - Sync Group Members
     - Public Channel or Private Channel
- Groups (Skipped)
- Members
     - Add a Member
     - Remove a Member
- Archive a channel

## Ticket Link

This PR fixes #7652 

## Checklist

- [x] This PR follows the Mattermost Contributing Guidelines.
- [x] Relevant product screenshots in the Denim theme have been added.
- [x]  Workflows are structured as a numbered list for the steps instead of paragraphs.
- [x] The changes are tested using a local build.

### CODE REVIEW

**Comment by Rutam21** [comment]
@cwarnermm Please review and suggest changes, if any. Thanks.

File: source/configure/user-management-configuration-settings.rst [hunk 1]
```diff
@@ -285,17 +285,189 @@ Channels
...
+2. Select the channel from the list and tap on it to view its configuration page.
+3. In the **Create Posts** section under **Advanced Access Control** tab, tick the checkbox for either **Guests** or **Members** or both to enable those users to post reactions in the channel.
+4. Click on **Save** to save the changes.
+
+.. image:: ../images/allow-post-reactions-for-a-channel.png
+  :alt: Add Members and Guests to post reactions to the channel using the System Console.
+
+Manage Members
+^^^^^^^^^^^^^^
+The ability for members to add and remove people.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:354]:
```suggestion
The ability for members to add and remove people from the channels. Guests can't add or remove people from channels.
```

File: source/configure/user-management-configuration-settings.rst [hunk 2]
```diff
@@ -285,17 +285,189 @@ Channels
...
+
+1. Go to **System Console > User Management > Channels** to access all available channels.
+2. Select the channel from the list and tap on it to view its configuration page.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:357]:
```suggestion
2. Select a channel from the list to view its configuration page.
```

File: source/configure/user-management-configuration-settings.rst [hunk 3]
```diff
@@ -285,17 +285,189 @@ Channels
...
+3. In the **Manage Members** section under **Advanced Access Control** tab, tick the checkbox for **Members** to enable those users to manage members for the channel.
+4. Click on **Save** to save the changes.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:359]:
```suggestion
4. Select **Save**.
```

File: source/configure/user-management-configuration-settings.rst [hunk 4]
```diff
@@ -285,17 +285,189 @@ Channels
...
+  - The ability to manage members for the channel is available for **Members** only. **Guests** can't add or remove members from the channel.
+
+Channel Mentions
+^^^^^^^^^^^^^^^^
+The ability for members and guests to use **@all**, **@here** and **@channel**.
+
+1. Go to **System Console > User Management > Channels** to access all available channels.
+2. Select the channel from the list and tap on it to view its configuration page.
+3. In the **Channel Mentions** section under **Advanced Access Control** tab, tick the checkbox for either **Guests** or **Members** or both to enable those users to use the available channel mentions.
+4. Click on **Save** to save the changes.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:374]:
```suggestion
4. Select **Save**.
```

File: source/configure/user-management-configuration-settings.rst [hunk 5]
```diff
@@ -285,17 +285,189 @@ Channels
...
+
+.. image:: ../images/allow-mentions-for-a-channel.png
+  :alt: Add Members and Guests to use mentions in a channel using the System Console.
+
+.. tip::                                                                                                                                                                
+  - **Guests** and **Members** can not use channel mentions without the ability to **Create Posts**. To enable this permission, these users must have been granted **Create Posts** permission first.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:380]:
```suggestion
  **Guests** and **Members** can't use channel mentions without the ability to **Create Posts**. To enable this permission, these users must have been granted **Create Posts** permission first.
```

File: source/configure/user-management-configuration-settings.rst [hunk 6]
```diff
@@ -285,17 +285,189 @@ Channels
...
+
+Manage Bookmarks
+^^^^^^^^^^^^^^^^
+The ability for members and guests to add, delete and sort bookmarks.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:384]:
```suggestion
The ability for members to add, delete, and sort bookmarks. Guests can't add, remove, or sort bookmarks for the channel.
```

File: source/configure/user-management-configuration-settings.rst [hunk 7]
```diff
@@ -285,17 +285,189 @@ Channels
...
+
+1. Go to **System Console > User Management > Channels** to access all available channels.
+2. Select the channel from the list and tap on it to view its configuration page.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:387]:
```suggestion
2. Select the channel from the list to view its configuration page.
```

File: source/configure/user-management-configuration-settings.rst [hunk 8]
```diff
@@ -285,17 +285,189 @@ Channels
...
+3. In the **Manage Bookmarks** section under **Advanced Access Control** tab, tick the checkbox for **Members** to enable those users to add, sort or remove bookmarks for the channel.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:388]:
```suggestion
3. In the **Manage Bookmarks** section under the **Advanced Access Control** tab, select **Members** to enable those users to add, sort, or remove bookmarks for the channel.
```

File: source/configure/user-management-configuration-settings.rst [hunk 9]
```diff
@@ -285,17 +285,189 @@ Channels
...
+4. Click on **Save** to save the changes.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:389]:
```suggestion
4. Select **Save**.
```

File: source/configure/user-management-configuration-settings.rst [hunk 10]
```diff
@@ -285,17 +285,189 @@ Channels
...
+
+.. image:: ../images/allow-manage-bookmarks-for-a-channel.png
+  :alt: Add Members to manage bookmarks for the channel using the System Console.
+
+.. tip::                                                                                                                                                                
+  - The ability to manage bookmarks for the channel is available for **Members** only. **Guests** can't add, remove or sort bookmarks for the channel.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:395]:
```suggestion
  The ability to manage bookmarks for the channel is available for **Members** only. **Guests** can't add, remove or sort bookmarks for the channel.
```

Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:395]:
```suggestion
```

File: source/configure/user-management-configuration-settings.rst [hunk 11]
```diff
@@ -285,17 +285,189 @@ Channels
...
+
+Channel Management
+~~~~~~~~~~~~~~~~~~
+
+Choose between inviting members manually or syncing members automatically from groups.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:400]:
```suggestion
Choose between inviting members manually or sychronizing members automatically from groups.
```

File: source/configure/user-management-configuration-settings.rst [hunk 12]
```diff
@@ -285,17 +285,189 @@ Channels
...
+
+Sync Group Members
+^^^^^^^^^^^^^^^^^^
+When enabled, adding and removing users from groups will add or remove them from this team. The only way of inviting members to this team is by adding the groups they belong to. See the :ref:`Synchronize teams and channels <onboard/ad-ldap-groups-synchronization:synchronize teams and channels>` documentation for further details.
+
+1. Go to **System Console > User Management > Channels** to access all available channels.
+2. Select the channel from the list and tap on it to view its configuration page.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:407]:
```suggestion
2. Select the channel from the list to view its configuration page.
```

File: source/configure/user-management-configuration-settings.rst [hunk 13]
```diff
@@ -285,17 +285,189 @@ Channels
...
+3. In the **Channel Management** tab, enable the **Sync Group Members** option.
+4. Click on **Save** to save the changes.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:409]:
```suggestion
4. Select **Save**.
```

File: source/configure/user-management-configuration-settings.rst [hunk 14]
```diff
@@ -285,17 +285,189 @@ Channels
...
+
+.. image:: ../images/sync-group-members-in-a-channel.png
+  :alt: Enable Sync Group Members for a channel using the System Console.
+
+Public channel or private channel
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If `public` the channel is discoverable and any user can join, or if `private` invitations are required to join the channel.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:416]:
```suggestion
Public channels are discoverable and any user can join. Private channels require invitations to join.
```

File: source/configure/user-management-configuration-settings.rst [hunk 15]
```diff
@@ -285,17 +285,189 @@ Channels
...
+
+1. Go to **System Console > User Management > Channels** to access all available channels.
+2. Select the channel from the list and tap on it to view its configuration page.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:419]:
```suggestion
2. Select the channel from the list to view its configuration page.
```

File: source/configure/user-management-configuration-settings.rst [hunk 16]
```diff
@@ -285,17 +285,189 @@ Channels
...
+3. In the **Public channel or private channel** section under **Channel Management** tab, toggle between **Public** or **Private** to change the channel visibility.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:420]:
```suggestion
3. In the **Public channel or private channel** section under the **Channel Management** tab, toggle between **Public** or **Private** to change channel visibility.
```

File: source/configure/user-management-configuration-settings.rst [hunk 17]
```diff
@@ -285,17 +285,189 @@ Channels
...
+4. Click on **Save** to save the changes.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:421]:
```suggestion
4. Select **Save**.
```

File: source/configure/user-management-configuration-settings.rst [hunk 18]
```diff
@@ -285,17 +285,189 @@ Channels
...
+Choose between inviting members manually or syncing members automatically from groups.
+
+Add members
+^^^^^^^^^^^
+
+1. Go to **System Console > User Management > Channels** to access all available channels.
+2. Select the channel from the list and tap on it to view its configuration page.
+3. In the **Members** section, tap on **Add Members**.
+4. Use the **Search** box to find a specific user or group to add to the channel.
+5. Click on **Add** to add the user or group as a **Member**.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:441]:
```suggestion
5. Select **Add** to add the user or group as a **Member**.
```

File: source/configure/user-management-configuration-settings.rst [hunk 19]
```diff
@@ -285,17 +285,189 @@ Channels
...
+6. Click on **Save** to save the changes.
+
+.. image:: ../images/add-members-to-a-channel.png
+  :alt: Add a member to the channel using the System Console.
+
+Remove members
+^^^^^^^^^^^^^^
+
+1. Go to **System Console > User Management > Channels** to access all available channels.
+2. Select the channel from the list and tap on it to view its configuration page.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:451]:
```suggestion
2. Select the channel from the list to view its configuration page.
```

File: source/configure/user-management-configuration-settings.rst [hunk 20]
```diff
@@ -285,17 +285,189 @@ Channels
...
+3. In the **Members** section, tap on **Remove** for the user that you want to remove.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:452]:
```suggestion
3. In the **Members** section, select **Remove** for the user that you want to remove.
```

File: source/configure/user-management-configuration-settings.rst [hunk 21]
```diff
@@ -285,17 +285,189 @@ Channels
...
+4. Click on **Save** to save the changes.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:453]:
```suggestion
4. Select **Save**.
```

File: source/configure/user-management-configuration-settings.rst [hunk 22]
```diff
@@ -285,17 +285,189 @@ Channels
...
+
+.. image:: ../images/remove-members-from-a-channel.png
+  :alt: Remove a member from the channel using the System Console.
+
+Archive a channel
+~~~~~~~~~~~~~~~~
+
+1. Go to **System Console > User Management > Channels** to access all available channels.
+2. Select the channel from the list and tap on it to view its configuration page.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:462]:
```suggestion
2. Select the channel from the list to view its configuration page.
```

File: source/configure/user-management-configuration-settings.rst [hunk 23]
```diff
@@ -285,17 +285,189 @@ Channels
...
+3. In the **Channel Profile** section, tap on **Archive Channel**.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:463]:
```suggestion
3. In the **Channel Profile** section, select **Archive Channel**.
```

File: source/configure/user-management-configuration-settings.rst [hunk 24]
```diff
@@ -285,17 +285,189 @@ Channels
...
+4. Click on **Save** to save the changes.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:464]:
```suggestion
4. Select **Save**.
```

File: source/configure/user-management-configuration-settings.rst [hunk 25]
```diff
@@ -285,17 +285,189 @@ Channels
...
+
+.. image:: ../images/archive-a-channel.png
+  :alt: Archive a channel using the System Console.
+
+.. tip::                                                                                                                                                                
+  - Channels can be deleted with all content, including posts in the database, using the :ref:`mmctl channel delete <manage/mmctl-command-line-tool:mmctl channel delete>` tool.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:470]:
```suggestion
  Channels can be deleted with all content, including posts in the database, using the :ref:`mmctl channel delete <manage/mmctl-command-line-tool:mmctl channel delete>` command.
```

File: source/configure/user-management-configuration-settings.rst [hunk 26]
```diff
@@ -285,17 +285,189 @@ Channels
...
+1. Go to **System Console > User Management > Channels** to access all available channels.
+2. Type the name of the channel in the **Search** box to find the particular channel, if it exists.
+
+.. image:: ../images/find-channels.png
+  :alt: Find a Mattermost channel using the System Console.
+
+Filter channel searches
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can filter System Console channel search to narrow down results based on the channel types (Public, Private or Archived), channel management types (Group Sync or Manual Invites) or the team that they belong to.
```
[the remark this answers is not in the document]
Comment (Rutam21) [reply source/configure/user-management-configuration-settings.rst]: Changes applied. Thanks.

**Comment by Rutam21** [comment]
@cwarnermm Thanks for the review. All the requested changes have been applied, and the PR is ready for your evaluation.

File: source/configure/user-management-configuration-settings.rst [hunk 27]
```diff
@@ -285,17 +285,186 @@ Channels
...
-| **Notes**:                                                                                                                            |
-|                                                                                                                                       |
-| - Channels can be deleted with all content, including posts in the database, using the                                                |
-|   :ref:`mmctl channel delete <manage/mmctl-command-line-tool:mmctl channel delete>` tool.                                             |
-| - You can search for channels by channel name or by channel ID.                                                                       |
-+-------------------------------------------------------------------------+-------------------------------------------------------------+
+Find Channels
+~~~~~~~~~~~~~
+
+You can find existing channels in your Mattermost instance using the System Console.
```
Comment (cwarnermm) [inline source/configure/user-management-configuration-settings.rst:291]:
```suggestion
Find existing channels in your Mattermost instance using the System Console.
```

**Review by cwarnermm — APPROVED** [review]
Thanks, @Rutam21!

**Pull request merged**